web/src/stream_data.ts

```diff
@@ -800,6 +800,32 @@ export function rewire_can_post_messages_in_stream(
     can_post_messages_in_stream = value;
 }
 
+export function user_can_move_messages_within_channel(stream_id: number): boolean {
```

Comment:
Most similar functions in this file accept a `sub: StreamSubscription` instead of an ID; should this one do that too?